src/sentry/reprocessing2.py

```diff
@@ -560,7 +569,7 @@ def start_group_reprocessing(
     # Later the activity is migrated to the new group where it is used to serve
     # the success message.
     new_activity = models.Activity.objects.create(
-        type=models.ActivityType.REPROCESS.value,
+        type=models.ActivityType.REPROCESS.value,  # type: ignore[attr-defined]
```

Comment:
this is concerning actually. is there no typing for enums in mypy?

Reply:
yeah this needs to be imported directly -- `from sentry.types.activity import ActivityType` -- I thought I had fixed all of these but it seems a few have creeped in

Reply:
more explicit import is required. updated it.